src/figuresClasses.ts

```diff
@@ -1,19 +1,74 @@
-export interface Figure {
+type Shapes = 'triangle'|'circle'|'rectangle';
+type Colors = 'red'|'green'|'blue';
 
+export interface Figure {
+  shape: Shapes;
+  color: Colors;
+  getArea: object;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: Shapes = 'triangle';
 
-}
+  constructor(
+    public color: Colors,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    if (a <= 0
+      || b <= 0
+      || c <= 0
+      || a + c <= b
+      || a + b <= c
+      || b + c <= a) {
+      throw new Error('Неправильн0!');
```

Comment:
usually we write error messages in english only

```suggestion
      throw new Error('your error message in english');
```